yamls/proto_nd_flow/resources/GeometryMC.yaml

```diff
@@ -3,12 +3,12 @@ path: proto_nd_flow.resources.geometry
 params:
   path: 'geometry_info'
   det_geometry_file: 'data/proto_nd_flow/2x2.yaml'
-  crs_geometry_files: ['data/proto_nd_flow/multi_tile_layout-2.4.16_v4.yaml',
-                       'data/proto_nd_flow/multi_tile_layout-2.5.16_v4.yaml']
+  crs_geometry_files: ['data/proto_nd_flow/multi_tile_layout-2.4.16.yaml',
```

Comment:
Was this reversion in crs geometry intentional? I think we still want _v4?

Reply:
this was unintentionally committed, I didn't have v4 locally and I can't see it in the [`/data/proto_nd_flow/` directory](https://github.com/DUNE/ndlar_flow/tree/develop/data/proto_nd_flow) - this change was just so I could run on the fly

Reply:
I've [pushed a reverted version](https://github.com/DUNE/ndlar_flow/pull/187/commits/ad53426ae11f66f824fe4aa23949a1848023221d) now